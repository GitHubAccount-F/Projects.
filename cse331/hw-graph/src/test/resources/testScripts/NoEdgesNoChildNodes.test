#Tests to that a node with no edges has no
#child node(s). Also tests that this applies
#to any nodes that are child nodes of other nodes.

#Creates empty graphs
CreateGraph graph1
CreateGraph graph2
CreateGraph graph3
CreateGraph graph4

#Adds nodes
AddNode graph1 one
AddNode graph2 one
AddNode graph2 two
AddNode graph3 one
AddNode graph3 two
AddNode graph3 three
AddNode graph4 one
AddNode graph4 two

#Create an edge
AddEdge graph4 one two e1

#Checks to see the child node
ListChildren graph1 one
ListChildren graph2 one
ListChildren graph2 two
ListChildren graph3 one
ListChildren graph3 two
ListChildren graph3 three
ListChildren graph4 one
ListChildren graph4 two