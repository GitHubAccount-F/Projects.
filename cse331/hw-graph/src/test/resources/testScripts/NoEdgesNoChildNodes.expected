#Tests to that a node with no edges has no
#child node(s). Also tests that this applies
#to any nodes that are child nodes of other nodes.

#Creates empty graphs
created graph graph1
created graph graph2
created graph graph3
created graph graph4

#Adds nodes
added node one to graph1
added node one to graph2
added node two to graph2
added node one to graph3
added node two to graph3
added node three to graph3
added node one to graph4
added node two to graph4

#Create an edge
added edge e1 from one to two in graph4

#Checks to see the child node
the children of one in graph1 are:
the children of one in graph2 are:
the children of two in graph2 are:
the children of one in graph3 are:
the children of two in graph3 are:
the children of three in graph3 are:
the children of one in graph4 are: two(e1)
the children of two in graph4 are: